src/util/get-date.util.ts

```diff
@@ -0,0 +1,12 @@
+import { Reference } from "../model/reference.model";
+
+/**
+ * Given a reference as parameter, returns a JS Date object
+ */
+export function getDate (ref: Reference): Date {
+    let raw = ref.issued["date-parts"][0];
+    let res: Date = new Date();
```

Comment:
For variables that don't change you should use `const`, as it can avoid some mistakes.